Gruntfile.js

```diff
@@ -275,6 +275,7 @@ module.exports = function(grunt) {
 					[ WORKING_DIR + 'wp-admin/js/plugin-install.js' ]: [ './src/js/_enqueues/admin/plugin-install.js' ],
 					[ WORKING_DIR + 'wp-admin/js/post.js' ]: [ './src/js/_enqueues/admin/post.js' ],
 					[ WORKING_DIR + 'wp-admin/js/postbox.js' ]: [ './src/js/_enqueues/admin/postbox.js' ],
+					[ WORKING_DIR + 'wp-admin/js/password-toggle.js' ]: [ './src/js/_enqueues/admin/password-toggle.js' ],
```

Comment:
Minor nitpick, could this be moved directly after `wp-admin/js/password-strength-meter.js`, to line 275, for alphabetical order?

Reply:
Should have realized these were alphabetical. I'll reorder.